solarforecastarbiter/datamodel.py

```diff
@@ -714,6 +718,59 @@ def __post_init__(self):
         __site_or_agg__(self)
 
 
+@dataclass(frozen=True)
+class EventForecast(Forecast):
+    """
+    Extends Forecast dataclass to include event forecast attributes.
+
+    Parameters
+    ----------
+    name : str
+        Name of the Forecast
+    issue_time_of_day : datetime.time
+        The time of day that a forecast run is issued, e.g. 00:30. For
+        forecast runs issued multiple times within one day (e.g. hourly),
+        this specifies the first issue time of day. Additional issue times
+        are uniquely determined by the first issue time and the run length &
+        issue frequency attribute.
+    lead_time_to_start : pandas.Timedelta
+        The difference between the issue time and the start of the first
+        forecast interval, e.g. 1 hour.
+    interval_length : pandas.Timedelta
+        The length of time between consecutive data points, e.g. 5 minutes,
+        1 hour.
+    run_length : pandas.Timedelta
+        The total length of a single issued forecast run, e.g. 1 hour.
+        To enforce a continuous, non-overlapping sequence, this is equal
+        to the forecast run issue frequency.
+    interval_label : str
+        Indicates if a time labels the beginning or the ending of an interval
+        average, or indicates an instantaneous value, e.g. beginning, ending,
+        instant.
```

Comment:
update

Reply:
Revising to add "event" as valid `interval_label` value.

Reply:
I thought the interval label had to be "event"

Reply:
Wondering if we need a post_init check for this too.

Reply:
Would an event forecast ever not have 'event' as the `interval_label`?